src/hooks/useComputed.ts

```diff
@@ -1,5 +1,6 @@
 import { autorun, IReactionDisposer } from "mobx";
 import { useMemo, useRef, useState } from "react";
+import equal from "fast-deep-equal";
```

Comment:
Even though we already have it in `node_modules`, can you add `fast-deep-equal` to `package.json` just to make it explicit that our code directly calls it?

Historically this didn't matter as much, but newer bundlers like vite can get confused w/o this being correct.